packages/animations/browser/test/shared.ts

```diff
@@ -13,16 +13,22 @@ import {buildAnimationAst} from '../src/dsl/animation_ast_builder';
 import {AnimationTrigger, buildTrigger} from '../src/dsl/animation_trigger';
 import {NoopAnimationStyleNormalizer} from '../src/dsl/style_normalization/animation_style_normalizer';
 import {triggerParsingFailed} from '../src/error_helpers';
+import {triggerParsingWarnings} from '../src/warning_helpers';
+import {RuntimeWarning} from '../src/warnings';
 import {MockAnimationDriver} from '../testing/src/mock_animation_driver';
 
 export function makeTrigger(
     name: string, steps: any, skipErrors: boolean = false): AnimationTrigger {
   const driver = new MockAnimationDriver();
   const errors: Error[] = [];
+  const warnings: RuntimeWarning[] = [];
   const triggerData = trigger(name, steps);
-  const triggerAst = buildAnimationAst(driver, triggerData, errors) as TriggerAst;
+  const triggerAst = buildAnimationAst(driver, triggerData, errors, warnings) as TriggerAst;
   if (!skipErrors && errors.length) {
     throw triggerParsingFailed(name, errors);
   }
+  if ((typeof ngDevMode === 'undefined' || !!ngDevMode) && warnings.length) {
+    console.warn(triggerParsingWarnings(name, warnings));
+  }
```

Comment:
Similar case here. You can simplify this since we're not shipping error codes and we're only warning in dev mode. The `console.warn` call can be embedded in the warnings helpers.

```suggestion
  if (warnings.length) {
    triggerParsingWarnings(name, warnings);
  }
```